runtime/hydradx/src/migrations.rs

```diff
@@ -0,0 +1,25 @@
+use super::*;
+
+use frame_support::{traits::OnRuntimeUpgrade, weights::Weight};
+pub struct OnRuntimeUpgradeMigration;
+impl OnRuntimeUpgrade for OnRuntimeUpgradeMigration {
+	#[cfg(feature = "try-runtime")]
+	fn pre_upgrade() -> Result<(), &'static str> {
+		Ok(())
+	}
+
+	fn on_runtime_upgrade() -> Weight {
+		let mut weight: Weight = Weight::zero();
+
+		frame_support::log::info!("Migrate Uniques Pallet start");
+		weight = weight.saturating_add(pallet_uniques::migration::migrate_to_v1::<Runtime, _, Uniques>());
+		frame_support::log::info!("Migrate Uniques Pallet end");
+
+		weight
+	}
+
+	#[cfg(feature = "try-runtime")]
+	fn post_upgrade() -> Result<(), &'static str> {
+		Ok(())
```

Comment:
Isnt post upgrade to verify that upgrade does it what is supposed to ?  Can we add a check for uniques ?

Reply:
Yes, but unfortunately it's not implemented for `uniques`, `pre_upgrade` and `post_upgrade` implementations are missing.